### ISSUE TITLE
[BUG] Channel receive redeem problem

### ISSUE THREAD

**Issue by haer0248** [mitchwadair/tesjs#84]
**Describe the bug**
If the title, prompt, or user_input contains the symbol '&', there is no way to redeem and receive the data.

**To Reproduce**
In the 'user_input' field, if you enter a query string URL like:
example.com?qs=1 -> success
example.com?qs=1&qs=2 -> failed.

**Expected behavior**

**Screenshots**
![image](https://github.com/mitchwadair/tesjs/assets/25150003/50514222-026d-4c66-89e6-69aa693127b6)

**Environment:**
- OS: Ubuntu 22.04.3 LTS
- Node Version [e.g. 15.3.0]: v16.14.0
- TESjs Version [e.g. 0.1.0]: 1.0.3

**Additional context**
I have opened the debug mode, and it displays:

BUG:
```
Thu, 14 Dec 2023 12:55:06 GMT - TESjs - Verifying webhook request
Thu, 14 Dec 2023 12:55:06 GMT - TESjs - Request contains message signature, calculating verification signature
Thu, 14 Dec 2023 12:55:06 GMT - TESjs - Request message signature 7347c8f00563c55a4f68122e0296291ecce8882312f017f62aebd4f65dedd7dd does not match calculated signature 7dd65c659958415a207b21f890bbeb5deb207d7a7186b5f99228e888eb102fde
```

Success:
```
Thu, 14 Dec 2023 13:02:30 GMT - TESjs - Verifying webhook request
Thu, 14 Dec 2023 13:02:30 GMT - TESjs - Request contains message signature, calculating verification signature
Thu, 14 Dec 2023 13:02:30 GMT - TESjs - Request message signature match
Thu, 14 Dec 2023 13:02:30 GMT - TESjs - Received notification for type channel.channel_points_custom_reward_redemption.add
```

**Comment by mitchwadair** [comment]
Thank you for the report! I'll look into this ASAP

**Comment by mitchwadair** [comment]
This should be fixed in `v1.0.4`. Let me know if you still have any issues. Thanks again for the report!
